### DISCUSSION TITLE
Is there a size limit for file transfers?

### DISCUSSION THREAD

**Discussion by xdroop** [pglombardo/PasswordPusher#871] [Q&A] [answered]
/t

**Comment by pglombardo** [comment, marked as answer]
Hi @xdroop - for privately hosted instances, there is no file size limit imposed currently.  For pwpush.com there are safeties I put in place until I decide a strategy.  (File uploads are still in Beta)

**Comment by ivancarlosti** [comment]
Hi @pglombardo , please how could we add a file size limit in hosted instances? :)

**Reply by pglombardo** [reply]
Unfortunately not currently.  I hope to revisit file uploads again soon.  Adding this is not much work.  I'll track the request in #2074 and update there once I have something.
